packages/suite/src/utils/wallet/cardanoUtils.ts

```diff
@@ -0,0 +1,262 @@
+import {
+    CARDANO,
+    CardanoAddressType,
+    CardanoCertificate,
+    CardanoCertificateType,
+    CardanoOutput,
+} from 'trezor-connect';
+import type { types } from '@fivebinaries/coin-selection';
+import { amountToSatoshi, formatAmount, formatNetworkAmount } from '@wallet-utils/accountUtils';
+import { Account } from '@wallet-types';
+import {
+    Output,
+    PrecomposedTransactionFinal,
+    PrecomposedTransactionFinalCardano,
+} from '@wallet-types/sendForm';
+import BigNumber from 'bignumber.js';
+import { PoolsResponse, StakePool } from '@suite/types/wallet/cardanoStaking';
+import { CARDANO_DEFAULT_TTL } from '@wallet-constants/sendForm';
+
+export const loadCardanoLib = async () => {
+    const lib = await import(
+        /* webpackChunkName: "@fivebinaries/coin-selection" */ '@fivebinaries/coin-selection'
+    );
+    return lib;
+};
+
+export const getProtocolMagic = (accountSymbol: Account['symbol']) =>
+    accountSymbol === 'ada' ? CARDANO.PROTOCOL_MAGICS.mainnet : 1097911063;
+
+export const getNetworkId = (accountSymbol: Account['symbol']) =>
+    accountSymbol === 'ada' ? CARDANO.NETWORK_IDS.mainnet : CARDANO.NETWORK_IDS.testnet;
+
+export const getAddressType = (accountType: Account['accountType']) =>
+    accountType === 'normal' ? CardanoAddressType.BASE : CardanoAddressType.BYRON;
+
+export const getStakingPath = (
+    accountType: Account['accountType'],
+    accountIndex: Account['index'],
+) => `m/${accountType === 'normal' ? 1852 : 44}'/1815'/${accountIndex}'/2/0`;
```

Comment:
there can't be any other account type in cardano than `normal` right? does bip43 purpose field value 44 make sense here?

Reply:
yup, there is only one account type since we've decided to not support byron accounts.